### PR TITLE
musicbox 0.1.6.5 (new formula)

### PR DESCRIPTION
**Pull request by darknessomi**
A sexy command line interface musicbox

### CODE REVIEW

File: Library/Formula/musicbox.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class MusicBox < Formula
+  desc "A sexy command line interface musicbox"
```
Comment (dunn) [inline Library/Formula/musicbox.rb]: Please don't describe software as "sexy".

File: Library/Formula/musicbox.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  homepage "https://github.com/darknessomi/musicbox"
+  url "https://pypi.python.org/packages/source/N/NetEase-MusicBox/NetEase-MusicBox-0.1.6.5.tar.gz#md5=f731f7c94804a42d2acde2825dba37e1"
+  version "0.1.6.5"
+  sha256 "8383d773bd1d4f5e1f4b41897fe34e3dac25c47d3f1b59f1cef62c35ce14a07e"
+  sha1 "d2f8cf412ba4a963ed76ec4dee2dc6fcd34a5652"
+
+  depends_on :mpg123 
+
+  def install
+    system "sudo", "python", "setup.py", "install"
```
Comment (dunn) [inline Library/Formula/musicbox.rb]: We have `*Language::Python.setup_install_args` available to use here; see [`ansible.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/ansible.rb) for an example.

File: Library/Formula/musicbox.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class MusicBox < Formula
+  desc "A concise command line interface musicbox"
+  homepage "https://github.com/darknessomi/musicbox"
+  url "https://pypi.python.org/packages/source/N/NetEase-MusicBox/NetEase-MusicBox-0.1.6.5.tar.gz#md5=f731f7c94804a42d2acde2825dba37e1"
+  version "0.1.6.5"
+  sha256 "8383d773bd1d4f5e1f4b41897fe34e3dac25c47d3f1b59f1cef62c35ce14a07e"
+  sha1 "d2f8cf412ba4a963ed76ec4dee2dc6fcd34a5652"
+
+  depends_on :mpg123
```
Comment (DomT4) [inline Library/Formula/musicbox.rb]: Should be `depends_on "mpg123"`. We only use `:xyz` for special dependencies, whereas `mpg123` is a normal dep.

File: Library/Formula/musicbox.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+class MusicBox < Formula
+  desc "A concise command line interface musicbox"
+  homepage "https://github.com/darknessomi/musicbox"
+  url "https://pypi.python.org/packages/source/N/NetEase-MusicBox/NetEase-MusicBox-0.1.6.5.tar.gz#md5=f731f7c94804a42d2acde2825dba37e1"
+  version "0.1.6.5"
+  sha256 "8383d773bd1d4f5e1f4b41897fe34e3dac25c47d3f1b59f1cef62c35ce14a07e"
+  sha1 "d2f8cf412ba4a963ed76ec4dee2dc6fcd34a5652"
```
Comment (DomT4) [inline Library/Formula/musicbox.rb]: Don't need a `sha1` with the `sha256`. Keep the SHA256 only.

File: Library/Formula/musicbox.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class MusicBox < Formula
+  desc "A concise command line interface musicbox"
+  homepage "https://github.com/darknessomi/musicbox"
+  url "https://pypi.python.org/packages/source/N/NetEase-MusicBox/NetEase-MusicBox-0.1.6.5.tar.gz#md5=f731f7c94804a42d2acde2825dba37e1"
```
Comment (DomT4) [inline Library/Formula/musicbox.rb]: Please cut the `#md5=f731f7c94804a42d2acde2825dba37e1` bit off the URL. At that point it should understand the version as well.

File: Library/Formula/musicbox.rb [hunk 6]
```diff
@@ -0,0 +1,51 @@
+class MusicBox < Formula
```
Comment (DomT4) [inline Library/Formula/musicbox.rb]: The `class` name here is wrong. It should be `class Musicbox < Formula`. We only use the second capital in the name to indicate there's a `-` in the name, such as `apache-ducks` would be `class ApacheDucks < Formula`.

File: Library/Formula/musicbox.rb [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+  
+  test do
+    system "musicbox", "--version"
+  end
+
```
Comment (DomT4) [inline Library/Formula/musicbox.rb]: Kill the blank line here.

**Comment by DomT4** [comment]
If you could possibly squash your commits back down to one after fixing the couple of nits highlighted that'd be great, Thanks!

**Comment by darknessomi** [comment]
@DomT4 Done! Thx!

**Pull request closed without merging**